package.json

```diff
@@ -2,22 +2,24 @@
   "name": "redux-persist",
   "version": "5.6.9",
   "description": "persist and rehydrate redux stores",
-  "main": "lib/index.js",
-  "module": "es/index.js",
+  "main": "dist/redux-persist",
+  "module": "dist/redux-persist.es",
   "types": "./src/index.d.ts",
   "repository": "rt2zz/redux-persist",
   "files": [
     "es",
     "src",
-    "lib"
+    "lib",
+    "dist"
   ],
+  "sideEffects": false,
   "scripts": {
     "ava": "BABEL_ENV=commonjs ava",
-    "build": "npm run flow-copy && npm run build:commonjs && npm run build:es && npm run build:umd && npm run build:umd:min",
+    "prebuild": "npm run clean",
+    "build": "rollup -c",
+    "build:old": "npm run flow-copy && npm run build:commonjs && npm run build:es && npm run build:umd && npm run build:umd:min",
```

Comment:
im not entirely sure what's proper way of supporting typings after going flat, "typings" from `flow-copy` will stop to work properly, I'd like to explore `gen-flow-files` tool (its flow builtin, but ive never used it)

TS will require some work, but at least its a sideway thing (have to be written manually anyway - so it doesnt really much, im concerned about flow because I'd like to generate them automatically and aint sure yet how to do it in a new setup)

Reply:
I am not sure either. gen-flow-files may work, last I tried it was buggy but that was maybe a year ago.

Figuring this out will be a big concern however, having types is important and maintaining separate flow types is not desirable obviously (maintaining .ts declarations is already a big burden).

Is it possible to bundle w/o stripping flow types? then we could basically ship a mirrored `redux-persist.flow.js` bundle, which is basically all flow-copy-source does anyway.

Reply:
unfortunately aint sure, need to play with those things first